modules/storage.js

```diff
@@ -0,0 +1,78 @@
+import Books from './books.js';
+
+const titleInput = document.getElementById('title');
+const authorInput = document.getElementById('author');
+const BookInfo = document.getElementById('booksContainer');
+const titreSect1 = document.getElementById('title-sec1');
+const addBookButton = document.getElementById('add-book');
+let BL = '';
+export const getmylibrary = JSON.parse(localStorage.getItem('mesLivres1'));
+
+export function renderBook() {
+  BookInfo.innerHTML = '';
+  getmylibrary.forEach((element, index) => {
+    const cardBody = document.createElement('tr');
+
+    const title = document.createElement('td');
```

Comment:
- [ ] Kindly fix the following error that appears on the console which causes the app to stop functioning properly.
![image](https://user-images.githubusercontent.com/62290545/171232861-69bea617-d099-40fc-a33a-9c74e1130b65.png)

Reply:
Hey @masangana, this error only occurs if your local storage didn't have the key you're looking for. That means if it's not there it will return as `null` which you're running the `forEach` on which causes the error.

Reply:
Thanks I just created the page for my project now i can find it

Reply:
@masangana, The very first time I open the demo (there's no key in the local storage at that point yet), I still get the error. However, after refreshing, the error disappears. You can try it yourself, just reset your local storage and see for yourself.

Reply:
An easy solution to this problem is to write something like the following: 
```
const getmylibrary = JSON.parse(localStorage.getItem('mesLivres1')) || []
```
which returns an empty array if there's no key in the local storage. Good luck.